src/typescript/azure-openapi-validator/rules/Rpaas_CreateOperationAsyncResponseValidation.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from "../rule"
+export const Rpaas_CreateOperationAsyncResponseValidation: string = "Rpaas_CreateOperationAsyncResponseValidation"
+
+rules.push({
+  id: "R4011",
+  name: Rpaas_CreateOperationAsyncResponseValidation,
+  severity: "error",
+  category: "ARMViolation",
```

Comment:
can you create a new category for the rpass rules ?

Reply:
sure.